Makefile

```diff
@@ -18,17 +18,20 @@ REGISTRY ?= andyzhangx
 REGISTRY_NAME = $(shell echo $(REGISTRY) | sed "s/.azurecr.io//g")
 IMAGE_NAME ?= smb-csi
 IMAGE_VERSION ?= v0.7.0
+WIN_IMAGE_VERSION ?= ${IMAGE_VERSION}-windows-1809-amd64

```

Comment:
`v0.7.0` tag does not work on GKE Windows node? I think you could use one version tag for both Linux and Windows.